src/api/public/apidocs-new/OBS-v2.10.50.yaml

```diff
@@ -8,7 +8,18 @@ tags:
   - name: Groups
 
 info:
-  description: Open Build Service API
+  description: |
+    The _Open Build Service API_ is a XML API.
+    This documentation is based on the _OpenAPI 3.0.0_ specification and powered by _Swagger_.
+
+    To authenticate, use [HTTP basic authentication](https://en.wikipedia.org/wiki/Basic_access_authentication) by passing the _Authorization_ header in the form of `Authorization: Basic <credentials>`.
+
+    We don't have any versioning.
+
+    We don't have rate limiting, so be conscious about how you use the API.
+    We do monitor the API, so abuse will lead to a warning, then a ban if it continues.
+
+    For command-line users, we recommend using [osc](https://github.com/openSUSE/osc) with its _api_ command to interact with the API.
```

Comment:
what is "its _api_" ? another one?

Reply:
My intention is to point users to `osc api`. I use the same terms as the manpage for `osc` and the output of `osc --help`, which are both listing `api` as one of the commands available for `osc`. I could add an example to make it clearer. Would this help?

Reply:
ah, misreaded it, sorry. Yes, an example would help and maybe point out that only the path part is needed unlike with curl. An easy example could be

 osc api /about